apps/base/migrations/0003_auto_20160216_2212.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
```

Comment:
i think these are automatically generated when you run `./manage.py makemigrations` and probably shouldn't be pushed.
#migrationcomment